src/_repobee/git/__init__.py

```diff
@@ -0,0 +1,28 @@
+"""Wrapper functions for git commands.
+
+.. module:: git
+    :synopsis: Wrapper functions for git CLI commands, such as push and clone.
+
+.. moduleauthor:: Simon Larsén
+"""
+
+from _repobee.git._push import PushSpec, push, push_async  # NOQA
```

Comment:
This should not be exposed in the API of this package, as it was private in the module.

```suggestion
from _repobee.git._push import PushSpec, push # NOQA
```